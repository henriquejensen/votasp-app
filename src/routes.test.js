import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Routes from './routes';

describe('Footer', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(( <Routes /> )))).toMatchSnapshot();
  })

  it('should match a snapshot', () => {
    expect(toJson(shallow(( <Routes /> )))).toMatchSnapshot();
  })
})
