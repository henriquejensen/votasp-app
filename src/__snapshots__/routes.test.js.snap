// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match a snapshot 1`] = `
<Fragment>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/como-funciona"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/app/redefinepassword"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/questionario-finalizado"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/perfil/:id"
  />
  <Route
    exact={true}
    path="/app"
    render={[Function]}
  />
  <Route
    path="/app/questionario/:question"
    render={[Function]}
  />
  <Route
    exact={true}
    path="/app/ranking"
    render={[Function]}
  />
  <Route
    exact={true}
    path="/app/atualizar-informacoes"
    render={[Function]}
  />
  <Route
    path="/app/candidatos-e-candidatas"
    render={[Function]}
  />
</Fragment>
`;

exports[`Footer should match a snapshot 2`] = `
<Fragment>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/como-funciona"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/app/redefinepassword"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/questionario-finalizado"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/perfil/:id"
  />
  <Route
    exact={true}
    path="/app"
    render={[Function]}
  />
  <Route
    path="/app/questionario/:question"
    render={[Function]}
  />
  <Route
    exact={true}
    path="/app/ranking"
    render={[Function]}
  />
  <Route
    exact={true}
    path="/app/atualizar-informacoes"
    render={[Function]}
  />
  <Route
    path="/app/candidatos-e-candidatas"
    render={[Function]}
  />
</Fragment>
`;
